webapp/src/main/webapp/vue-apps/space-form/components/SpaceFormDrawer.vue

```diff
@@ -21,215 +21,226 @@
-            </v-stepper-content>
-          </template>
-          <template v-if="includeInvitation">
-            <v-stepper-step
-              v-if="!singleStep"
-              :step="invitationStep"
-              class="ma-4 pa-0"
-              editable>
-              {{ $t('spacesList.label.inviteUsers') }}
-            </v-stepper-step>
-            <v-stepper-content :step="invitationStep" class="pa-0 ma-0 no-border">
-              <space-form-invitation
-                v-if="stepper === invitationStep"
-                @invited-members="space.invitedMembers = $event"
-                @invited-email="space.externalInvitedUsers = $event" />
-            </v-stepper-content>
-          </template>
-          <template v-if="includeAccess">
-            <v-stepper-step
-              v-if="!singleStep"
-              :step="accessStep"
-              class="ma-4 pa-0"
-              editable>
-              {{ $t('spacesList.label.spaceAccessTitle') }}
-            </v-stepper-step>
-            <v-stepper-content :step="accessStep" class="pa-0 ma-0 no-border">
-              <space-form-access
-                v-if="stepper === accessStep"
-                :visibility="space.visibility"
-                :subscription="space.subscription"
-                @visibility="space.visibility = $event"
-                @subscription="space.subscription = $event" />
-            </v-stepper-content>
-          </template>
-        </v-stepper>
-      </v-expand-transition>
-    </template>
-    <template v-if="drawer && (spaceTemplate || isEdit)" slot="footer">
-      <div class="d-flex">
-        <v-btn
-          v-if="stepper > 1 && !isEdit"
-          class="btn"
-          @click="previousStep">
-          {{ $t('spacesList.button.back') }}
-        </v-btn>
-        <v-spacer />
-        <v-btn
-          :disabled="savingSpace || spaceSaved"
-          class="btn me-2"
-          @click="cancel">
-          <template>
-            {{ $t('spacesList.button.cancel') }}
-          </template>
-        </v-btn>
-        <v-btn
-          v-if="stepper < lastStep"
-          class="btn btn-primary"
-          @click="nextStep">
-          {{ $t('spacesList.button.next') }}
-        </v-btn>
-        <v-btn
-          v-else
-          :loading="savingSpace"
-          :disabled="saveButtonDisabled"
-          class="btn btn-primary"
-          @click="saveSpace">
-          <v-icon v-if="spaceSaved">mdi-check-all</v-icon>
-          <template v-else-if="isEdit">
-            {{ $t('spacesList.button.updateSpace') }}
-          </template>
-          <template v-else>
-            {{ $t('spacesList.button.createSpace') }}
-          </template>
-        </v-btn>
+      </template>
+    </exo-drawer>
+    <div class="d-none d-lg-block position-absolute">
+    <space-preview
+      v-if="drawer && spaceTemplate"
+      :space="space"
+      :preview-avatar="previewAvatar"
+      class="pa-4 z-index-modal"
+      style="top: 6vh; left: 55vw;"/>
```

Comment:
please ad it inside the drawer content and format the file using `npm run lint` command line